pkg/base1/Makefile.am

```diff
@@ -16,6 +16,8 @@ nodist_base_DATA = \
 	pkg/base1/jquery.min.js.gz \
 	pkg/base1/cockpit.min.js.gz \
 	pkg/base1/bundle.min.js.gz \
+	pkg/base1/react.min.js.gz \
+	pkg/base1/react-dom.min.js.gz \
```

Comment:
The split is an annoying react choice. But will do a follow up replacing this with react-lite, so not an issue.